src/main/java/ch/njol/skript/expressions/ExprLevel.java

```diff
@@ -121,6 +121,8 @@ public void change(final Event e, final @Nullable Object[] delta, final ChangeMo
 					assert false;
 					continue;
 			}
+			if (level < 0)
+				continue;
```

Comment:
Do you think it would be better to just treat negative inputs as 0?

Reply:
IMO this might give unexpected behavior if someone e.g. accidentally added a minus sign or some calculation went wrong, so skipping it would be the least affecting action